patient-view-be/src/main/java/spvproject/patientviewbe/model/ChiefComplaintModel.java

```diff
@@ -0,0 +1,66 @@
+package spvproject.patientviewbe.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class ChiefComplaintModel {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@OneToOne
+	private CodeModel code;
+
+	@Column
+	private String editedBy;
+
+	@Column
+	private String admisionNote;
+
+	public ChiefComplaintModel() {
+		
+	}
+
+	public ChiefComplaintModel(int id, CodeModel code, String editedBy, String admisionNote) {
```

Comment:
This can simply be named `ChiefComplaint` as we already know it is a model, since it is inside the `/model/` folder